File: Interface/frutNatura_funci2/app/src/main/java/com/frutnatura/app/api/LoginResponse.java
package com.frutnatura.app.api;

import com.google.gson.annotations.SerializedName;

public class LoginResponse {
    @SerializedName("success")
    public boolean sucesso;
    @SerializedName("error")
    public String mensagem;
    @SerializedName("accessToken")
    public String token;
    @SerializedName("usuarioId")
    public String usuarioId;
    @SerializedName("name")
    public String nome;
    @SerializedName("role")
    public String role;
}

File: Desktop/Models/LoginResponse.cs
using System;
using System.Text.Json.Serialization;

namespace FrutNatura.Desktop.Models
{
    public sealed class LoginResponse
    {
        [JsonPropertyName("accessToken")]
        public string? AccessToken { get; set; }

        [JsonPropertyName("success")]
        public bool Success { get; set; }

        [JsonPropertyName("error")]
        public string? Error { get; set; }

        [JsonPropertyName("refreshToken")]
        public string? RefreshToken { get; set; }

        [JsonPropertyName("usuarioId")]
        public Guid UsuarioId { get; set; }

        [JsonPropertyName("name")]
        public string? Name { get; set; }

        [JsonPropertyName("role")]
        public string? Role { get; set; }
    }
}
